src/ui/server/shared-validation/website/index.test.ts

```diff
@@ -0,0 +1,67 @@
+import validateWebsite from '.';
+import generateValidationErrors from '../../helpers/validation';
+
+describe('shared-validation/website', () => {
+  const errors = {};
+
+  describe('when errors', () => {
```

Comment:
Can we be consistent with descriptions, in other places it's labelled "with errors / without errors"

Reply:
changed to with / without errors